mmv1/products/cgc/terraform.yaml

```diff
@@ -230,6 +230,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+     - !ruby/object:Provider::Terraform::Examples
```

Comment:
I think this block is off by one space which unfortunately matters for .yaml files